student.rb

```diff
@@ -5,7 +5,14 @@ def initialize(_clasroom, age, name = 'unknown', parent_permission: true)
     @classroom = classroom
```

Comment:
- [ ] Kindly fix your syntax error here. Your `initialize` method parameter contains `clasroom` but you are creating a `@classroom` instance variable from `classroom`, which you did not define.

  ![yaseer-class](https://user-images.githubusercontent.com/36542986/151337148-6d4086e7-4488-41b6-807e-d8f8d74f1f34.jpg)